src/renderer/components/mulmo_viewer/media_player_dummy.vue

```
@@ -0,0 +1,42 @@
+<template>
+  <div>
+    <div v-if="videoWithAudioSource">
+      <video
+        :src="videoWithAudioSource"
+        class="mx-auto h-auto max-h-[80vh] w-auto object-contain"
+        :controls="controlsEnabled"
+        ref="videoWithAudioRef"
```

ESLint

[error] 'videoWithAudioRef' is defined as ref, but never used.

(vue/no-unused-refs)